src/kOS.Safe.Test/ListValueTest.cs

```diff
@@ -62,17 +62,16 @@ public void CanGetIndex()
             var length = InvokeDelegate(list, "LENGTH");
             Assert.AreEqual(4,length);
 
-            IIndexable indexable = list;
```

Comment:
These are changed because the ListValue class correctly implements IList directly now so 

    var list = new ListValue();
    list[0] = foo;

and the rest of basic list stuff you can now do directly.